modules/tinymce/CHANGELOG.md

```diff
@@ -59,6 +59,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - The `editor.annotator.remove` did not keep selection when removing the annotation #TINY-8195
 
 ### Removed
+- Removed the old `plugin-tinycomments-tinymce4` skin from Oxide #TINY-8409
```

Comment:
This was never bundled as part of the TinyMCE package so we shouldn't mention it here.
```suggestion
```